Ladybird/Android/src/main/cpp/LadybirdActivity.cpp

```diff
@@ -76,10 +83,20 @@ Java_org_serenityos_ladybird_LadybirdActivity_initNativeCode(JNIEnv* env, jobjec
     Core::EventLoopManager::install(*event_loop_manager);
     s_main_event_loop = make<Core::EventLoop>();
 
+    // The arguments cannot be empty
+    Vector<StringView> strings;
+    strings.append("ladybird"sv);
+    char argv[1][9] = { { 'l', 'a', 'd', 'y', 'b', 'i', 'r', 'd', '\0' } };
+    Main::Arguments arguments = {
+        .argc = 1,
+        .argv = (char**)argv,
+        .strings = strings
+    };
```

Comment:
Is it truly the case that args and argv must have a value in order to be parsed by ArgsParser?

i.e., does

```
Main::Arguments {
    .argc = 0,
    .argv = nullptr
    .strings = {}
};
```
Not work?

If it's truly the case that at least one `strings` element is required, then we should be able to get away with only passing that as a span, because Serenity's shell does this:

```
    Vector<StringView> arguments;
    TRY(arguments.try_ensure_capacity(command.argv.size()));
    for (auto& arg : command.argv)
        arguments.unchecked_append(arg);

    Main::Arguments arguments_object {
        .argc = 0,
        .argv = nullptr,
        .strings = arguments
    };
```

When processing shell builtins.
    
At which point a dummy arguments of
    
```
    Main::Arguments arguments {
        .argc = 0,
        .argv = nullptr,
        .strings = ReadonlySpan { "ladybird"sv }
    };
```
"should work"

Reply:
If the strings are empty, the program exits thinking that a parse error had happened.
The argv and argc can be set to 0 and nullptr.
I couldn't use ReadonlySpan because the compiler would complain about it being const, so I used Span